### PR TITLE
[APM-PHP] improve installation instructions for who installs from tar.gz

### PR DESCRIPTION
**Pull request by labbati**
### What does this PR do?
This PR improves step by step instructions to install the APM PHP extension from tar.gz in two scenarios: using the automatic installation script and using manual process.

### Motivation
We had report of users being confused about the steps to do in order to have a fully manual installation in case of tar.gz, including the correct PHP extension version to be chosen for every PHP version. This PR adds such information and improve the overall instructions.

### Preview
[Preview](https://docs-staging.datadoghq.com/apm/php/php-extension-apis/tracing/faq/php-tracer-manual-installation/#install-from-the-targz-archive)

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/tracing/faq/php-tracer-manual-installation.md [hunk 1]
```diff
@@ -15,10 +15,63 @@ From the location you downloaded the package:
 
 ```bash
 $ tar -xf datadog-php-tracer.tar.gz -C /
```
Comment (SammyK) [inline content/en/tracing/faq/php-tracer-manual-installation.md:17]:
```suggestion
$ tar -xf datadog-php-tracer.tar.gz -C
```

**Review by SammyK — COMMENTED** [review]
Other than a tweak to the `tar` command, this LGTM. Thanks @labbati! 👍

File: content/en/tracing/faq/php-tracer-manual-installation.md (hunk 1 above)
Comment (labbati) [reply content/en/tracing/faq/php-tracer-manual-installation.md:17]: @SammyK the final `/` is necessary if we keep using `-C` ([docs](https://linux.die.net/man/1/tar)). did you mean also to replace `-C` with some other flag?

Comment (SammyK) [reply content/en/tracing/faq/php-tracer-manual-installation.md:17]: Oh no, I didn't read the command correctly. 😞 Please ignore this suggestion. Sorry for the noise!

Comment (labbati) [reply content/en/tracing/faq/php-tracer-manual-installation.md:17]: ahahah thanks Sammy, sounds good.

**Review by SammyK — APPROVED** [review]

**Review by apigirl — APPROVED** [review]

**Pull request merged**